docs/docs/configuration/sql-templating.mdx

```diff
@@ -17,8 +17,8 @@ made available in the Jinja context:
 
 - `columns`: columns which to group by in the query
 - `filter`: filters applied in the query
-- `from_dttm`: start `datetime` value from the selected time range (`None` if undefined)
-- `to_dttm`: end `datetime` value from the selected time range (`None` if undefined)
+- `from_dttm`: start `datetime` value from the selected time range (`None` if undefined) (deprecated, use `get_time_filter` instead)
```

Comment:
Maybe say "deprecated beginning in version 4.1.0" since we don't have versioned docs?

Reply:
Good call - I'll deprecate as of 5.0, as this will likely not be cherried into the 4.1 branch.